doc/source/flexibility/compressed_columns.rst

```diff
@@ -267,8 +267,8 @@ options to do this safely:
 Version Specific Information
 ============================
 
-  * :rn:`5.7.17-11`
-    Feature implemented in |Percona Server| 5.7
+  * :rn:`8.0.12-1`
+    Feature ported from |Percona Server| 5.7.
 
```

Comment:
AFAIK this isn't ported (yet) so shouldn't be part of the docs

Reply:
Resolving by adding the *Availability* field right at the top of the section. The currently available version has an *Important* block. But the site has lost special formatting for it and it does not look important at all.